package.json

```diff
@@ -210,6 +211,7 @@
     "lodash": "4.17.15",
     "log4js": "6.1.0",
     "marksy": "8.0.0",
+    "mathjs": "^10.6.4",
```

Comment:
I see this added here, but is it ever imported?

Reply:
Whoops, no it isn't -- this was for the Insights page, I forgot to remove it when I split the PRs. Good catch!